generate-routes.ts

```diff
@@ -1,13 +1,33 @@
 import { readFile, writeFile } from 'node:fs/promises'
-import { resolve } from 'node:path'
+import { dirname, resolve } from 'node:path'
+import { fileURLToPath } from 'node:url'
 
 import { openapi } from '@seamapi/types/connect'
 import { camelCase, kebabCase, pascalCase, snakeCase } from 'change-case'
+import { deleteAsync } from 'del'
 import { ESLint } from 'eslint'
 import { format, resolveConfig } from 'prettier'
 
-const rootClassPath = resolve('src', 'lib', 'seam', 'connect', 'seam-http.ts')
-const routeOutputPath = resolve('src', 'lib', 'seam', 'connect', 'routes')
+const rootPath = resolve(
+  dirname(fileURLToPath(import.meta.url)),
+  'src',
+  'lib',
+  'seam',
+  'connect',
+)
+const rootClassPath = resolve(rootPath, 'seam-http.ts')
+const routeOutputPath = resolve(rootPath, 'routes')
+
+async function main(): Promise<void> {
+  const routes = createRoutes()
+  const routeNames = await Promise.all(routes.map(writeRoute))
+  const routeIndexName = await writeRoutesIndex(routes)
+  await deleteAsync([
```

Comment:
We can't do `deleteAsync(routeOutputPath)` or we get an error (due to an unexpected interaction with esloader)

<img width="915" alt="image" src="https://github.com/seamapi/javascript-http/assets/721372/27d5b3c9-e29c-43f8-9123-0f934b010973">